### PR TITLE
Added info about ability to overwrite

### PR DESCRIPTION
**Pull request by mr1716**
Added information that the processor can overwrite fields if the field already exists. The code already exists to do this, as its built in, but the documentation doesnt reflect this.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by bmorelli25** [comment]
Hi @mr1716, thanks for the contribution! Would you please sign the CLA in https://github.com/elastic/beats/pull/29944#issuecomment-1018549547.

File: libbeat/processors/actions/docs/add_fields.asciidoc [hunk 1]
```diff
@@ -6,8 +6,9 @@
 ++++
 
 The `add_fields` processor adds additional fields to the event.  Fields can be
-scalar values, arrays, dictionaries, or any nested combination of these.  By
-default the fields that you specify will be grouped under the `fields`
+scalar values, arrays, dictionaries, or any nested combination of these. 
+The add_fields processor does have the ability to overwrite taget field if it already exists.
```
Comment (bmorelli25) [inline libbeat/processors/actions/docs/add_fields.asciidoc:10]:
```suggestion
The `add_fields` processor will overwrite the target field if it already exists. 
```

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -14,6 +14,9 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Filebeat*
 
+==== Added
+Added description into add_fields to reflect that it can overwrite the desired field.
+
```
Comment (bmorelli25) [inline CHANGELOG.next.asciidoc:19]: We don't typically add changelog entries for documentation.

```suggestion
```

**Comment by mr1716** [comment]
> Hi @mr1716, thanks for the contribution! Would you please sign the CLA in [#29944 (comment)](https://github.com/elastic/beats/pull/29944#issuecomment-1018549547).

Hey, I signed this agreement on 1/21 in the afternoon.

**Comment by bmorelli25** [comment]
Thanks! I should've waited for the test to run before asking again. Looks like we're good.

**Comment by mr1716** [comment]
@bmorelli25 All good! Looks like we're close to this being complete!

**Comment by mr1716** [comment]
@bmorelli25 Any sense of when the final review could happen and this might get merged into the main branch?

**Review by dedemorton — APPROVED** [review]
LGTM. I'll take care of merging and backporting this PR when the build is green. Thank you for your contribution!

**Pull request merged**